Userland/Applications/Run/RunWindow.cpp

```diff
@@ -142,9 +142,9 @@ bool RunWindow::run_via_launch(DeprecatedString const& run_input)
     auto url = URL::create_with_url_or_path(run_input);
 
     if (url.scheme() == "file") {
-        auto real_path = Core::DeprecatedFile::real_path_for(url.path());
+        auto real_path = Core::FileSystem::real_path(url.path());
         if (real_path.is_null()) {
-            // errno *should* be preserved from Core::DeprecatedFile::real_path_for().
+            // errno *should* be preserved from Core::FileSystem::real_path().
```

Comment:
It's slightly out of scope for this commit, but while we are here anyways, we should probably make this a `// FIXME: Make real_path use ErrorOr<> and use that to get the error code`.

While we might be able to do that inside the implementation itself, we shouldn't be assuming that our more modern wrappers set or retain errno correctly to the outside.